code/src/styles/happythoughts.css

```diff
@@ -0,0 +1,38 @@
+.thoughts-box {
+  border: 1px solid #000;
+  box-shadow: 6px 6px #000;
+  width: 315px;
```

Comment:
Notice that the fixed width of thoughts-box and post-box makes them a bit to big when surfing on small screens like galaxy fold.